pages/common/forever.md

```diff
@@ -0,0 +1,15 @@
+# forever - nodejs tool
+
+> Server-side JavaScript application that makes sure node applications run indefinitely (restarts after crash).
+
+- Start running a file forever (as a daemon):
+
+`forever {{file}}.js`
+
+- List forever running processes:
```

Comment:
That sentence sounds a bit odd to me. Maybe `List running forever processes:` is a better fit?

Reply:
Well, if you say so, then the following would be best IMO:
> List running "forever" processes